src/Screens/Wishlist/Wishlist.js

```diff
@@ -1,7 +1,23 @@
-import React from "react";
+import React, { useEffect } from "react";
+import WishListCart from "../../Components/WishListCart/WishListCart";
+import { useWishList } from "../../Contexts";
+import "./Wishlist.css";
 
 const Wishlist = () => {
-  return <div>Wishlist</div>;
+  const {
+    state: { wishList },
+  } = useWishList();
+
+  return (
+    <div className="wishlist-container">
+      <h2>My wishlist</h2>
+      <div className="wishlist-products">
+        {wishList.map((product) => {
+          return <WishListCart product={product} />;
+        })}
+      </div>
+    </div>
```

Comment:
If the wishlist is empty, you can display a text saying that there are no items in the wishlist.

Reply:
Yes. I forgot it. Thanks